pavelib/prereqs.py

```diff
@@ -31,7 +31,11 @@
 
 # Developers can have private requirements, for local copies of github repos,
 # or favorite debugging tools, etc.
-PRIVATE_REQS = 'requirements/philu/base.txt'
+if 'TOXENV' in os.environ:
+    PRIVATE_REQS = 'requirements/philu/testing.txt'
+else:
+    PRIVATE_REQS = 'requirements/philu/base.txt'
+
```

Comment:
Can we use ternary operator here?

PRIVATE_REQS = 'requirements/philu/testing.txt' if 'TOXENV' in os.environ else 'requirements/philu/base.txt'

Reply:
@ahmed-faraz46

Reply:
@harris-irfan I feel like what is written is more readable, ternary operator sometimes improves readability but in this case if we see the line length will increase and condition is little complex `TOXENV in os.environ` due to which it is effecting the readability please have a look again keeping in mind the readability, if you still feel we should change it, we will.